h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -2719,7 +2721,7 @@ private void doCompute() {
       }
       if (_state._iter >= _parms._max_iterations)
         _job.warn("Reached maximum number of iterations " + _parms._max_iterations + "!");
-      if (_parms._nfolds > 1 && !Double.isNaN(_lambdaCVEstimate))
+      if (_parms._nfolds > 1 && !Double.isNaN(_lambdaCVEstimate) && _model._output._submodels.length > _bestCVSubmodel)
```

Comment:
Since we add the time limitation to the main model training it is possible that we won't have enough time to calculate through all the submodels that we need => fallback to the `pickBestModel` instead of failing with `ArrayIndexOutOfBounds`.

Reply:
sounds reasonable.
@wendycwong can you please confirm this shouldn't have any unexpected effect?